app/diagnostics.go

```diff
@@ -108,6 +108,15 @@ func pluginActivated(pluginStates map[string]*model.PluginState, pluginId string
 	return state.Enable
 }
 
+func pluginVersion(pluginsAvailable []*model.BundleInfo, pluginId string) string {
```

Comment:
Maybe use a more specific name, e.g. `getPluginVersionFromBundle`?

Reply:
I agree but was trying to keep consistency with the previous function `pluginActivated` hence the name `pluginVersion`. Thoughts @hanzei ?

Reply:
Good point :+1: